### ISSUE TITLE
Python 3 - Error with "decode('utf-8')

### ISSUE THREAD

**Issue by Izymask** [JazzCore/python-pdfkit#32]
Hello, 

There's an error  in Python 3 with the line (in the **init** method of PDFKit)

> self.wkhtmltopdf = self.configuration.wkhtmltopdf.decode('utf-8') 
> 'str' object has no attribute 'decode'

We had to remove the call of the decode function to make it work.

**Comment by jorge-marques** [comment]
I am having the same issue.
You can work around it by using a byte string.

`pdfkit.configuration(wkhtmltopdf=bytes('/path/to/wkhtmltopdf', 'utf-8'))`

**Comment by debaetsr** [comment]
pdfkit.py line 101:
```python
elif self.source.isFileObj():
            input = self.source.source.read().encode('utf-8')
```
also gives problems when the source is already in utf-8 (encoding utf-8 to utf-8 gives weird results). I don't use the from_string, but I guess the problem will also exists there. 

Removing the encode works for me.

I assume it's the programmers job to ensure correct encoding before calling the library, so he can be in complete control what to do if unsupported characters occur.

Regards,
Ruben

**Comment by thundergolfer** [comment]
I ran into this same problem with the following code:

```
path_wkthmltopdf = os.path.join(os.path.dirname(__file__),
                                    'wkhtmltopdf','bin','wkhtmltopdf.exe').encode('utf-8')
 # path_wkthmltopdf is a string
config = pdfkit.configuration(wkhtmltopdf=path_wkthmltopdf)
```

Why can't I pass a string filepath as an option? PdfKit expects a bytestring as thus the `AttributeError: 'str' object has no attribute 'decode'` error.

Is there a reason why it is this way that I do not see?

Cheers

**Comment by alanhamlett** [comment]
We should add a test to check specifically for this, but should be fixed now in current master.

**Comment by JazzCore** [comment]
Should be fixed in #72, closing
